From 7bc12d0202d5819442dd4c32755feb5eb19af70b Mon Sep 17 00:00:00 2001
From: Marc Hartmayer <mhartmay@linux.ibm.com>
Date: Wed, 8 Jan 2025 12:33:05 +0100
Subject: [PATCH] rust/pvimg: Document the change from '--comm-key' to '--cck'
 in the help message

This fixes problems when users search for '--comm-key' in the help
message.

Fixes: 5b6d7a467dc3 ("rust/pvimg: Add '--cck <FILE>' command line option and make '--comm-key' an alias")
Reviewed-by: Steffen Eiden <seiden@linux.ibm.com>
Reviewed-by: Nico Boehr <nrb@linux.ibm.com>
Signed-off-by: Marc Hartmayer <mhartmay@linux.ibm.com>
Signed-off-by: Steffen Eiden <seiden@linux.ibm.com>
---
 rust/pvimg/src/cli.rs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rust/pvimg/src/cli.rs b/rust/pvimg/src/cli.rs
index 446e9b74..f5a8c308 100644
--- a/rust/pvimg/src/cli.rs
+++ b/rust/pvimg/src/cli.rs
@@ -330,7 +330,8 @@ pub struct CreateBootImageArgs {
 
     /// Use the content of FILE as the customer-communication key (CCK).
     ///
-    /// The file must contain exactly 32 bytes of data.
+    /// The file must contain exactly 32 bytes of data. This option used to be
+    /// called '--comm-key' in previous versions.
     #[arg(long, value_name = "FILE", visible_alias = "comm-key")]
     pub cck: Option<PathBuf>,
 
